CHANGELOG.md

```diff
@@ -26,6 +26,28 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-0267][]:
+  
+  Non-generic members that support a generic parameter list, including nested type declarations, are now allowed to carry a contextual `where` clause against outer generic parameters. Previously, such declarations could only be expressed by placing the member inside a dedicated constrained extension.
+
+  ```swift
+  struct Box<Wrapped> {
+    func boxes() -> [Box<Wrapped.Element>] where Wrapped: Sequence { ... }
+  }
+  ```
+  Because contextual `where` clauses are effectively visibility constraints, overrides adopting this feature must be at least as visible as the overriden method:
```

Comment:
```suggestion
  Because contextual `where` clauses are effectively visibility constraints, overrides adopting this feature must be at least as visible as the overridden method:
```